AbletonMCP_Remote_Script/handlers/dispatch.py

```
@@ -0,0 +1,259 @@
+        if track_index < 0 or track_index >= len(song.return_tracks):
+            raise IndexError("Return track index out of range")
+        return song.return_tracks[track_index]
+    elif track_type == "master":
+        return song.master_track
+    else:
+        if track_index < 0 or track_index >= len(song.tracks):
+            raise IndexError("Track index out of range")
+        return song.tracks[track_index]
+
+
+def _get_track_routing(song, p, ctrl):
```

Ruff (0.15.1)

[warning] Unused function argument: `ctrl`

(ARG001)